lib/workers/repository/init/index.spec.ts

```diff
@@ -29,5 +33,16 @@ describe(getName(__filename), () => {
       const renovateConfig = await initRepo({});
       expect(renovateConfig).toMatchSnapshot();
     });
+    it('warns on unsupported options', async () => {
+      apis.initApis.mockResolvedValue({} as never);
+      onboarding.checkOnboardingBranch.mockResolvedValueOnce({});
+      config.getRepoConfig.mockResolvedValueOnce({
+        filterUnavailableUsers: true,
+      });
+      config.mergeRenovateConfig.mockResolvedValueOnce({});
+      secrets.applySecretsToConfig.mockReturnValueOnce({} as never);
+      await initRepo({});
+      expect(warn.mock.calls).toMatchSnapshot();
```

Comment:
```suggestion
      expect(logger.logger.warn.mock.calls).toMatchSnapshot();
```

Reply:
For some reason this errored for me locally. Let's see what CI thinks 😀

Reply:
Does [not work unfortunately](https://github.com/renovatebot/renovate/pull/9406/checks?check_run_id=2411334457#step:10:17):

```
$ tsc --noEmit
Error: lib/workers/repository/init/index.spec.ts(41,33): error TS2339: Property 'mock' does not exist on type '{ (msg: string): void; (meta: Record<string, any>, msg?: string): void; }'.
```

Any other ideas @viceice?

Reply:
Sorry, haven't seen https://github.com/renovatebot/renovate/pull/9406#discussion_r618482431.